bootstrap/kubeadm/types/v1beta1/types.go

```diff
@@ -227,11 +227,14 @@ type NodeRegistrationOptions struct {
 // Networking contains elements describing cluster's networking configuration
 type Networking struct {
 	// ServiceSubnet is the subnet used by k8s services. Defaults to "10.96.0.0/12".
-	ServiceSubnet string `json:"serviceSubnet"`
+	// +optional
+	ServiceSubnet string `json:"serviceSubnet,omitempty"`
 	// PodSubnet is the subnet used by pods.
-	PodSubnet string `json:"podSubnet"`
+	// +optional
```

Comment:
It looks like kubeadm will default ServiceSubnet and DNSDomain but not PodSubnet. Should we keep this one as required?